python/raft-ann-bench/src/raft-ann-bench/plot/__main__.py

```diff
@@ -397,6 +429,15 @@ def main():
                               algorithms",
         default=None,
     )
+    parser.add_argument(
+        "--groups",
+        help="plot only comma separated groups of parameters",
```

Comment:
Can you also update the algorithms help string to add that unless specified with 'groups' or 'algo-groups', the parameters in the 'base' group of the yml config are executed by default.